### PR TITLE
Fix typos and teach ReSharper new words

### PR DESCRIPTION
**Pull request by jnm2**

### CODE REVIEW

File: src/NUnitFramework/framework/Attributes/RandomAttribute.cs [hunk 1]
```diff
@@ -76,15 +76,15 @@ public RandomAttribute(uint min, uint max, int count)
         }
 
         /// <summary>
-        /// Construct a set of longs within a specified range
+        /// Generates <see cref="long"/> values within a specified range.
```
Comment (jnm2) [inline src/NUnitFramework/framework/Attributes/RandomAttribute.cs]: This makes the intellisense display language-agnostic.

File: src/NUnitFramework/framework/Attributes/RandomAttribute.cs [hunk 2]
```diff
@@ -647,9 +647,6 @@ public override IEnumerable GetData(IParameterInfo parameter)
 
         #region DecimalDataSource
 
-        // Currently, Randomizer doesn't implement methods for decimal
-        // so we use random Ulongs and convert them. This doesn't cover
-        // the full range of decimal, so it's temporary.
```
Comment (jnm2) [inline src/NUnitFramework/framework/Attributes/RandomAttribute.cs]: No longer true

File: src/NUnitFramework/tests/Internal/Filters/MockTestFilter.cs [hunk 3]
```diff
@@ -57,7 +57,7 @@ public enum MatchFunction
 
         /// <summary>
         /// The result of the match-function <see cref="_expectedFunctionToBeCalled"/>
-        /// for the teste <see cref="_expectedTest"/>.
+        /// for the test <see cref="_expectedTest"/>.
```
Comment (jnm2) [inline src/NUnitFramework/tests/Internal/Filters/MockTestFilter.cs]: Technically not a typo, but thankfully ReSharper was none the wiser :D

File: src/NUnitFramework/framework/Attributes/RangeAttribute.cs [hunk 4]
```diff
@@ -111,13 +111,13 @@ public RangeAttribute(long from, long to, long step)
         #region Unsigned Longs
 
         /// <summary>
-        /// Construct a range of unsigned longs using default step of 1
+        /// Constructs a range of <see cref="long"/> values using the default step of 1.
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Attributes/RangeAttribute.cs]:
```suggestion
        /// Constructs a range of <see cref="ulong"/> values using the default step of 1.
```

File: src/NUnitFramework/framework/Attributes/RangeAttribute.cs [hunk 5]
```diff
@@ -111,13 +111,13 @@ public RangeAttribute(long from, long to, long step)
...
         /// </summary>
         [CLSCompliant(false)]
         public RangeAttribute(ulong from, ulong to) : this(from, to, 1ul) { }
 
         /// <summary>
-        /// Construct a range of unsigned longs specifying the step size
+        /// Constructs a range of <see cref="long"/> values with the specified step size.
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Attributes/RangeAttribute.cs]:
```suggestion
        /// Constructs a range of <see cref="ulong"/> values with the specified step size.
```

File: src/NUnitFramework/framework/Compatibility/System.Collections.Concurrent/ConcurrentQueue.cs [hunk 6]
```diff
@@ -126,7 +126,7 @@ private void OnSerializing(StreamingContext context)
         }
 
         /// <summary>
-        /// Construct the queue from a previously seiralized one
+        /// Construct the queue from a previously serialized one
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Compatibility/System.Collections.Concurrent/ConcurrentQueue.cs]:
```suggestion
        /// Construct the queue from a previously serialized one.
```
For consistency 😄.

**Pull request merged**